src/FileConventions/Library.fs

```diff
@@ -410,32 +410,35 @@ let ProjFilesNamingConvention(fileInfo: FileInfo) =
 
     fileName <> parentDirectoryName
 
+let DoesNamespaceInclude (fileInfo: FileInfo) (word: string) =
+    let fileText = File.ReadLines fileInfo.FullName
+
+    if fileText.Any() then
+        let firstLine = fileText.First()
+
+        if firstLine.Contains "namespace" then
```

Comment:
just return `
firstLine.Contains "namespace" && firstLine.Contains word`

Reply:
@parhamsaremi shouldn't this actually be firstLine.StartsWith "namespace"?

Reply:
hmmmm maybe. I didn't change the logic I just thought this way it'll be cleaner. But I think StartsWith should work too.